src/style.css

```diff
@@ -1 +1,97 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Heavy.ttf') format('truetype');
+  font-weight: 700;
+  font-style: normal;
+}
+
+.paige {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.paige__body {
+  font-family: Avenir, sans-serif;
```

Comment:
Consider adding a fallback font-family. While 'Avenir' is specified, it's a good practice to provide a generic family in case the primary font is not available, for example, 'font-family: Avenir, sans-serif;'.